pytrip/lib/core.c

```diff
@@ -641,12 +644,62 @@ static PyObject * calculate_lvh_slice(PyObject *self, PyObject *args)
     return PyArray_Return(vec_out);
 }
 
+static void append_intersection(PyObject *list_out, double x, double y, double z){
+    PyObject *list_item; // temporary variable to store point in 3D space (represented as list of 3 floats)
+
+    list_item = PyList_New(3);
+
+    PyList_SetItem(list_item, 0, PyFloat_FromDouble(x));
+    PyList_SetItem(list_item, 1, PyFloat_FromDouble(y));
+    PyList_SetItem(list_item, 2, PyFloat_FromDouble(z));
+
+    PyList_Append(list_out, list_item);
+}
+
+static void calc_intersect_sagittal(PyObject *vec_slice, PyObject *list_out, double depth, int i_0, int i_1){
+    double x_0, x_1, y_0, y_1, z;
+    double a;
```

Comment:
rename a for fraction or factor, so it resembles linear interpolation

Reply:
i think it does:
![image](https://user-images.githubusercontent.com/44035398/149636900-b2acd8a4-4058-4678-a545-8a0c0f8d41c1.png)